### PR TITLE
DeltaMax's Taiko Oni Overlord Wiki Page

### PR DESCRIPTION
**Pull request by fajar13k**
The finished DMX 2 tournament wiki page.

---

### CODE REVIEW

File: wiki/Tournaments/DMX/2/en.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+  - **[sakuraburst - skyshifter vip (yea) \[Oni\]](/beatmapsets/747322/#taiko/1574742)**
+
+------------------------------------------------------------------------
+
+## Match Results
+
+### Grand Finals
+
+| Sunday, 2018-08-26 |  |  |  |  | 
+| --: | :-: | :-: | :-- | :-- | :-: |
```
Comment (Joehuu) [inline wiki/Tournaments/DMX/2/en.md]: Broken tables on every match result table. Remove one column on these lines.

File: wiki/Tournaments/DMX/2/en.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
+---
+tags:
+- DeltaMax's Taiko Oni Overlord
+- DMX 2
+---
+# DeltaMax's Taiko Oni Overlord
+
+![DMX 2 Logo](img/logo.png)
+
+The **DeltaMax's Taiko Oni Overlord (***DMX 2***)** is a double elimination one-on-one osu!taiko tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 2nd installment of the DeltaMax's Tournament Series.
```
Comment (Joehuu) [inline wiki/Tournaments/DMX/2/en.md]: `DeltaMax’s Tournament Series` or `DeltaMax’s Tournaments`?

File: wiki/Tournaments/DMX/2/en.md [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 months of osu!supporter tag |
+
+## Organisation
+
+The Taiko Suiji Cup is run by various osu!taiko community members.
```
Comment (Joehuu) [inline wiki/Tournaments/DMX/2/en.md]: `is run` -> `was run`

File: wiki/Tournaments/DMX/2/en.md [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+| **H** | ![][flag_BR] [Foxerus](/users/7479684) | ![][flag_GB] [Braixen-](/users/8628814) | ![][flag_HK] [Akali393394](/users/9686628) | ![][flag_BR] [AnonX32](/users/2730270) |
+
+-----------------------
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Team(s) |
+| --- | :-- |
```
Comment (Joehuu) [inline wiki/Tournaments/DMX/2/en.md]: Center crowns.

File: wiki/Tournaments/DMX/2/en.md [hunk 5]
```diff
@@ -0,0 +1,491 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 4 months of osu!supporter tag, unique profile badge |
+| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | 2 months of osu!supporter tag |
+| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | 1 months of osu!supporter tag |
+
+## Organisation
+
+The Taiko Suiji Cup was run by various osu!taiko community members.
```
Comment (Joehuu) [inline wiki/Tournaments/DMX/2/en.md]: `Taiko Suiji Cup` -> `DeltaMax's Taiko Oni Overlord`

File: wiki/Tournaments/DMX/2/en.md [hunk 6]
```diff
@@ -0,0 +1,491 @@
...
+
+## Groups
+
+| Group | Top Seed | High Seed | Low Seed | Unseeded |
+| :-: | :-- | :-- | :-- | :-- |
+| **A** | ![][flag_AT] [animexamera](/users/7511357) | ![][flag_IT] [Kith190](/users/7365988) | ![][flag_AU] [cactusthegreat](/users/5592409) | ![][flag_MY] [Bedwyr Aorta](/users/10875855) |
+| **B** | ![][flag_BR] [Mikudayo](/users/5235718) | ![][flag_US] [Konan](/users/3252800) | ![][flag_IT] [VenomPlus](/users/7722827) | ![][flag_EC] [Gamelan4](/users/9856910) |
+| **C** | ![][flag_FR] [Yoshi-GH](/users/3317042) | ![][flag_FR] [\_Infected\_](/users/7432095) | ![][flag_AU] [Beat43210](/users/5664171) | ![][flag_BR] [Omega Ranger](/users/3234628) |
+| **D** | ![][flag_PT] [hi im costarino](/users/9209430) | ![][flag_PL] [Wolfgang](/users/7184324) | ![][flag_GB] [-Ryan](/users/7822081) | ![][flag_US] [CaptainEChan](/users/9536977) |
+| **E** | ![][flag_MY] [cdhsausageboy](/users/2403621) | ![][flag_AU] [Tsubasa2](/users/6835183) | ![][flag_BR] [Lazarento](/users/3224958) | ![][flag_US] [Rukairi](/users/6642597) |
```
Comment (juankristal) [inline wiki/Tournaments/DMX/2/en.md]: Lazarento is noted as high seed on the previous table but low seed on this one.

**Pull request merged**